### DISCUSSION TITLE
Healing nerf reducing power of consumable items

### DISCUSSION THREAD

**Discussion by godnesso** [chromiecraft/chromiecraft#3240] [Suggestions]
The global healing nerf in raid content intends to reduce the effect of healing classes being more powerful than their Classic counterparts.
As a side effect, healing consumables, such as Healthstones, Healing Potions and Bandages are now reduced as well, even though the health pools of classes are roughly the same and the power of these consumables has never changed (in some cases even reduced, i.e. potion cooldowns not beginning to reset until you are out of combat).

This can be very frustrating, especially as a melee player, when sometimes you cannot rely on healers to pay attention and your healing item is not saving you because of tuning making you take more damage and the item itself healing less than intended.
I know this has been solved on a previous PTE server by making a list for all healing spells utilized by these consumable items that excludes them from being reduced in power.

Thoughts?

**Comment by Icebeam666** [comment]
I think this is a great idea. I don't think stuff like healthstones, healing potions and bandages should be nerfed.

**Comment by Plasmashield** [comment]
This healing reduction just created a whole bunch of problems. And their fixes for some talents or pots appear not as quickly as we would like. It seems to me that the reduction was relevant in UBRS and may be in MC, but with the opening of BWL, I think that this should be removed for all raids

**Comment by KaneoHunter** [comment]
> Furthermore, bandages and healing potions will no longer be affected either.

Seems this suggestion has been acted upon, can Healthstones get the same love? Also the Divine Aegis talent for Disc, which double dips the nerf similarly to beacon.

**Reply by jeskej** [reply]
> > Furthermore, bandages and healing potions will no longer be affected either.
> 
> Seems this suggestion has been acted upon, can Healthstones get the same love?

Came here to suggest the same thing

**Comment by Hyperbearr** [comment]
Shameless bump because healthstones deserve justice!

**Comment by Nyeriah** [comment]
Healthstones as well as other consumables are no longer affected by the nerf. If you find any of them that still is, open a bug report about it.
